src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/RuntimeMetrics.cs

```diff
@@ -188,17 +187,16 @@ private static IEnumerable<Measurement<long>> GetGarbageCollectionCounts()
             }
         }
 
-        // TODO - Uncomment once an implementation for https://github.com/dotnet/runtime/issues/104844 is available.
-        //private static IEnumerable<Measurement<double>> GetCpuTime()
-        //{
-        //    if (OperatingSystem.IsBrowser() || OperatingSystem.IsTvOS() || OperatingSystem.IsIOS())
-        //        yield break;
+        private static IEnumerable<Measurement<double>> GetCpuTime()
+        {
+           if (OperatingSystem.IsBrowser() || OperatingSystem.IsTvOS() || OperatingSystem.IsIOS())
```

Comment:
```suggestion
           if (OperatingSystem.IsBrowser() || OperatingSystem.IsTvOS() || (OperatingSystem.IsIOS() && !OperatingSystem.IsMacCatalyst()))
```
OperatingSystem.IsIOS returns true for macCatalyst. (macCatalyst is a subset of iOS.)